recipes/eko/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "eko" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/N3PDF/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/eko-{{ version }}.tar.gz
```
We generally prefer PyPI as the source is slightly smaller. Since this is pure python, it's not a problem if you cannot run `pytest`.

---

Yes, the problem would be running pytest. Then is it fine to have no tests?